go-backend/db/core_value.go

```diff
@@ -0,0 +1,144 @@
+package db
+
+import (
+	"context"
+	"time"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+const (
+	listCoreValuesQuery    = `SELECT * FROM core_values WHERE org_id = $1 and parent_core_value_id IS NULL`
+	listSubCoreValuesQuery = `SELECT * FROM core_values WHERE org_id = $1 and parent_core_value_id = $2`
+	getCoreValueQuery      = `SELECT * FROM core_values WHERE org_id = $1 and id = $2`
+	createCoreValueQuery   = `INSERT INTO core_values (org_id, core_value_text,
+		description, parent_core_value_id, created_at, updated_at) VALUES ($1, $2, $3, $4, $5, $6) RETURNING id`
+	deleteSubCoreValueQuery = `DELETE FROM core_values WHERE org_id = $1 and parent_core_value_id = $2`
+	deleteCoreValueQuery    = `DELETE FROM core_values WHERE org_id = $1 and id = $2`
+	updateCoreValueQuery    = `UPDATE core_values SET (core_value_text, description, updated_at) =
+		($1, $2, $3) where id = $4 and org_id = $5`
+)
+
+type CoreValue struct {
+	ID                int64     `db:"id" json:"id"`
+	OrgID             int64     `db:"org_id" json:"org_id"`
+	CoreValueText     string    `db:"core_value_text" json:"core_value_text"`
```

Comment:
Struct name is already CoreValue, why repeat that in fields. This can be simply called as Text

Reply:
similar for any other attribute also

Reply:
we will be making this change in db schema too.
cc @sunil4sonawane